cypress/fixtures/todos.json

```diff
@@ -0,0 +1,1106 @@
+[
```

Comment:
You do not need the whole list for the test. I think a few todos will be enough.

Reply:
fixed